packages/experimental/Avatar/macos/AvatarViewManager.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import FluentUI
+
+@objc(MSFAvatarViewManager)
+class AvatarViewManager: RCTViewManager {
+	override func view()->NSView! {
+		let avatarView = AvatarView(avatarSize: 72)
+		return avatarView
+	}
+
+	override class func requiresMainQueueSetup() -> Bool {
+		return true
+	}
+
+	override func constantsToExport() -> [AnyHashable : Any]! {
+		return [
+			"sizes" : [
+				"xSmall" : 16,
+				"small" : 24,
+				"medium" : 32,
+				"large" : 40,
+				"xLarge" : 52,
+				"xxLarge" : 72
+			]
+		]
+	}
```

Comment:
API difference: 
iOS has these explicit sizes you can set the Avatar too. macOS supports any size. To keep the same API, I just hardcoded sizes for macOS avatar to use and call the constructor with the appropriate CGFloat,

Reply:
yeah, can we bring this up with the design team? Avatar for mac is still bit undefine but there are lots of churn i don't know Taili's team might have more guidance :)

Reply:
There's a figma somewhere with values I haven't looked up.. but that would be a FluentUI Apple change ideally. The purpose of this was really to unblock arm64 testing, so would it be cool to keep this enum here (perhaps with better defined constants) and integrate it back into FUA with the better design once we have it?

Reply:
I asked Kaan to start, and he said the point values will most likely stay the same. Nevertheless, I filed https://github.com/microsoft/fluentui-apple/issues/391 to track adding this to FluentUI Apple